usaspending_api/awards/management/commands/loadfinancialassistance.py

```diff
@@ -0,0 +1,189 @@
+import csv
+import logging
+from datetime import datetime
+
+import django
+from django.core.management.base import BaseCommand, CommandError
+
+import usaspending_api.awards.management.commands.helpers as h
+from usaspending_api.awards.models import Award, FinancialAssistanceAward
+from usaspending_api.common.threaded_data_loader import ThreadedDataLoader
+from usaspending_api.references.models import Agency, LegalEntity, Location
+from usaspending_api.submissions.models import SubmissionAttributes
+"""
+
+No PIID in the download from https://www.usaspending.gov/DownloadCenter/Pages/dataarchives.aspx ?
+
+"""
+
+
+class Command(BaseCommand):
+    help = "Loads awards from a usaspending financial assistance download. \
+            Usage: `python manage.py loadcontracts source_file_path`"
+
+    logger = logging.getLogger('console')
+
+    def add_arguments(self, parser):
+        parser.add_argument('file', nargs=1, help='the file to load')
+
+    def handle(self, *args, **options):
+        # Create a new submission attributes object for this timestamp
+
+        subattr = SubmissionAttributes()
+        subattr.usaspending_update = datetime.now()
+        subattr.save()
+
+        # field_map is used to simply translate a column name in our schema
+        # to the field name in the input source
+        field_map = {
+            "award_id": "federal_award_id",
+            "federal_action_obligation": "fed_funding_amount",
+            "non_federal_funding_amount": "non_fed_funding_amount",
+            "cfda_number": "cfda_program_num",
+            "cfda_title": "cfda_program_title",
+            "uri": "unique_transaction_id",
```

Comment:
Turns out that I've got a lot of records with no `fain` and blank `uri`.  I don't know what to use to call `Award.get_or_create_summary_award` with in that case.  If multiple FA records can be tied together into a single award, I don't see a field in the CSV to help me do that.  On the other hand, if each row in my source csv is going to have its own award, I could just generate and use a GUID for each row.